package.json

```diff
@@ -21,11 +21,11 @@
   "dependencies": {
     "@docusaurus/core": "2.4.3",
     "@docusaurus/plugin-content-docs": "2.4.3",
-    "@docusaurus/theme-common": "^2.4.3",
-    "@docusaurus/theme-mermaid": "^2.4.3",
+    "@docusaurus/theme-common": "2.4.3",
+    "@docusaurus/theme-mermaid": "2.4.3",
```

Comment:
We don't minor version resolution dedupe bumps? I guess they probably wouldn't happen in this dep tree. What is the reason for this change just curious?

Reply:
We need the docusaurus version to match the plugin versions so linking works properly with `docusaurus-openrpc`. and the docusaurus-openrpc has a dep on `@docusaurus/theme-common` as well.